Backend/app/auth/auth_service.py

```diff
@@ -43,21 +43,18 @@
 
 
 def create_access_token(data: dict[str, str], expires_delta: timedelta | None = None) -> str:
-    """Create a jwt token from data with a expire date
+    """Creates a JWT access token with given data and expiration.
```

Comment:
Period is optional for docs but I'm not against having them

Reply:
ok